packages/core-components/src/docs/getting-started/50-github-packages-installation-guide.stories.mdx

```diff
@@ -10,6 +10,25 @@ Visit [npm](https://www.npmjs.com/search?q=%40otto-de) and install our core comp
 npm i @otto-de/b2b-core-components @otto-de/b2b-tokens
 ```
 
+To use the components, you need to load a CSS file and some JavaScript. The CSS file includes the fonts and the design tokens.
```

Comment:
If the tokens are included in the CSS anyway, we don't need teams to install them separately if they are using the components package. 

We might want to include this in the docs so people know that the tokens package is no longer needed if they use the components too.